go-controller/pkg/node/openflow_manager.go

```diff
@@ -211,9 +211,15 @@ func (c *openflowManager) Run(stopChan <-chan struct{}, doneWg *sync.WaitGroup)
 }
 
 func (c *openflowManager) updateBridgePMTUDFlowCache(key string, ipAddrs []string) {
+	// protect defaultBridge config from being updated by gw.nodeIPManager
+	c.defaultBridge.Lock()
```

Comment:
my bad I missed this in my PR, can you make this a separate commit as I will need to include it in my backports downstream?